apps/docs/src/.vitepress/components/OnyxIconLibrary.vue

```diff
@@ -0,0 +1,47 @@
+<script lang="ts" setup>
+import { ICON_CATEGORIES } from "@sit-onyx/icons";
+import OnyxIcon from "../../../../../packages/sit-onyx/src/components/OnyxIcon/OnyxIcon.vue";
+
+const ALL_ICONS = import.meta.glob("../../../node_modules/@sit-onyx/icons/src/assets/*.svg", {
+  as: "raw",
+  eager: true,
+});
+
+const getIconContent = (iconName: string) => {
+  return ALL_ICONS[`../../../node_modules/@sit-onyx/icons/src/assets/${iconName}.svg`];
+};
+</script>
+
+<template>
+  <div>
+    <section v-for="(icons, category) in ICON_CATEGORIES" :key="category" class="category">
+      <h4 class="category__headline">{{ category }}</h4>
```

Comment:
<img width="336" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/93249943-e121-469f-832c-7a5933cdb008">

As the above headline is H2, best practice would be to make this h3 -> don't skip headline levels.
```suggestion
      <h3 class="category__headline">{{ category }}</h3>
```